ddl/ddl_api.go

```diff
@@ -1106,6 +1106,12 @@ func checkConstraintNames(constraints []*ast.Constraint) error {
 
 // checkInvisibleIndexOnPK check if primary key is invisible index.
 func checkInvisibleIndexOnPK(tblInfo *model.TableInfo) error {
+	// PKIsHandle == true means the table already has a visible primary key,
```

Comment:
Please move this comment above the signature of this function(L1107) so that it is more obvious to the possible developers in the future.

Reply:
@bb7133 addressed, PTAL